### ISSUE TITLE
It looks like there is an "Analyzing move..." BUG.

### ISSUE THREAD

**Issue by Chess321** [lightvector/KataGo#814]
Is KataGo limited to 5 GB RAM?
https://github.com/sanderland/katrain/issues/636

**Comment by lightvector** [comment]
No, KataGo shouldn't be limited to any particular amount of RAM. 

Here's a possible guess: KataGo has a limit on the maximum number of simultaneous positions to analyze, which the user or GUI program that is using it can freely configure. Presumably Katrain chooses some limit. Also, perhaps the way Katrain works, is that it actually attempts to *achieve* the number of visits you specify on every position of the game. I.e. maybe you should treat the number of visits not as a maximum, but as an actual value that it will run until it reaches. Since you set an extremely large number, it is busy attempting to reach that number on the first N positions of the game, and as soon as you reach the limit of the number of simultaneous positions to analyze, the later positions in the game don't get any analysis because they have to wait for the earlier ones to finish.

I don't know if Katrain actually works this way. I don't know if this has anything to do whatsoever with the real problem. KataGo does have a way for the GUI program to cancel earlier queries rather than waiting for them to finish - I don't know if Katrain uses that feature or not. Maybe you can read through the settings and documentation and Katrain and figure out whether something like this is likely to be the case, or if it is some other problem of this flavor, and whether you can correct it with the settings. So far, it doesn't sound to me that there is a bug in KataGo.

(By the way, you might also consider upgrading the neural net you are using, the "g170e-b15c192" net is many years old and far weaker than current neural nets. The latest ones are available at https://katagotraining.org/networks ).
